crates/spfs/src/storage/fs/tag.rs

```diff
@@ -883,6 +923,23 @@ impl TagLock {
             }
         }
     }
+
+    /// Remove the lock file for a tag
+    ///
+    /// # Safety:
+    /// Tag locks are used to ensure that only one process is writing to a tag file at a time.
+    /// Removing the lock file without ensuring that the tag file is not being written to may
+    /// cause the data within the file to become corrupt.
+    pub unsafe fn remove<P: AsRef<Path>>(tag_file: P) -> Result<()> {
+        let mut lock_file = tag_file.as_ref().to_path_buf();
+        lock_file.set_extension("tag.lock");
```

Comment:
Now that this `"tag.lock"` appears in multiple places I suggest moving the value to a const.